src/icons/fill/Dashboard.jsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
```

Comment:
shouldn't the copyright year will be 2022?

Reply:
yep, updating